### PR TITLE
trial

### PR DESCRIPTION
**Pull request by snehamariamthomas**
Hi Yazmin!
Hope this works

### CODE REVIEW

File: MY470_wk5_class.ipynb [hunk 1]
```diff
@@ -152,7 +152,8 @@
    "outputs": [],
    "source": [
     "# Exercise 3: Open a new issue in your partner's repository. \n",
-    "\n"
+    "\n",
+    "print(\"Hi Yazmin!\")\n"
```
Comment (yazbap) [inline MY470_wk5_class.ipynb:156]: I don't know about this? how about we do something else!

**Review by yazbap — COMMENTED** [review]
here is a summary of all the comments theoretically

File: MY470_wk5_class.ipynb (hunk 1 above)
Comment (snehamariamthomas) [reply MY470_wk5_class.ipynb:156]: Okay then hi anonymous?

Comment (yazbap) [reply MY470_wk5_class.ipynb:156]: okay!

**Pull request merged**